ramls/copycatprofile.json

```diff
@@ -51,6 +69,9 @@
     }
   },
   "additionalProperties": false,
-  "required": [ "name" ]
-
+  "required": [
+    "name",
+    "createJobProfileId",
+    "updateJobProfileId"
+  ]
```

Comment:
Stricly speaking this is a breaking change. I think we can be practical here and say the profile was not worth if didn't include these.